index.html

```diff
@@ -0,0 +1,128 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta http-equiv="X-UA-Compatible" content="IE=edge">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/5.15.4/css/all.min.css" />
+  <link rel="stylesheet" href="styles/style.css">
+  <link href="https://fonts.googleapis.com/icon?family=Material+Icons" rel="stylesheet">
+  <title>capstone</title>
+</head>
+<body>
+
+<div class="beforeNav">
+  <ul class="beforeNavUl">
+    <li class="beforeNavLi">
+      <a href="https://web.facebook.com/alaminrakib.tonoy"><i class="social fab fa-facebook-f"></i></a>
+    </li>
+    <li class="beforeNavLi"><a href="https://twitter.com/elonmusk"><i class="social fab fa-twitter"></i></a></li>
+    <li class="beforeNavLi"><a href="#" class="beforeNavLink">English</a></li>
+    <li class="beforeNavLi"><a href="#" class="beforeNavLink">Logout</a></li>
+  </ul>
+</div>
+<header class="header">
+  <a href="index.html"><img class="logo" src="image/icons/qawaliLogo.png" alt="logo" /></a>
+  <nav class="nav">
+    <ul class="navUl">
+      <li class="navLi"><a href="#headline" class="navLink">Home</a></li>
+      <li class="navLi"><a href="#programs" class="navLink">Program</a></li>
+      <li class="navLi"><a href="#performers" class="navLink">performers</a></li>
+      <li class="navLi"><a href="about.html" class="navLink">About</a></li>
+    </ul>
+  </nav>
```

Comment:
- [ ] Please ensure that the home page has a link to the About section. so on this page, the link should be a link to the about page.

Reply:
@MrBrN197 
Thank you, honorable code reviewer. After your valuable feedback I have managed to change the following:

- Home page has a link to the About section now.

Reply:
@MrBrN197 
May I submit it now, for code review again?